packages/browser-wallet/src/background/recovery.ts

```diff
@@ -128,11 +153,15 @@ async function performRecovery({ providers, ...recoveryInputs }: Payload) {
     }
     await addIdentity(identitiesToAdd);
     await addCredential(credsToAdd);
+    await sessionIsRecovering.set(false);
```

Comment:
If some error happens during the recovery method, then this will not be set to `false` as far as I can see. So if I open the wallet, then what happens? It will show the spinner but it will never complete?

Reply:
Yes. :fearful: